### ISSUE TITLE
Would like API access to collection

### ISSUE THREAD

**Issue by jgoldshlag** [fafranco82/swdestinydb#120]
I'm thinking of making an iOS app to track my collection, and would like to be able to sync it to swdestinydb. It looks like you have API methods for decks, but not for my collection. Any chance of getting this?

**Comment by azezpz1** [comment]
Yeah, this would be great. I was looking into trying to make an android app, and was hoping a user would be able to view and edit their collection in the app.

**Comment by nux** [comment]
I also was just looking into this. Would be nice simply sync my collection between my wife's account and my own.

**Comment by nux** [comment]
I happen to primarily work in symfony and have built plenty of APIs. If I submitted a pull request would it be considered?

**Comment by fafranco82** [comment]
Due to my limited time, all PRs are welcome ;)
